web/src/components/Calendar.js

```
@@ -300,28 +331,24 @@ const calendarContainer = css`
 `
 
 const dayBox = css`
-  margin-bottom: ${theme.spacing.md};
+  margin-bottom: ${theme.spacing.xs};
   display: flex;
+  justify-content: space-between;
+  align-items: center;
 
   .day-box {
-    font-size: ${theme.font.lg};
-    width: 11em;
-    display: inline-block;
-    border: 2px solid ${theme.colour.grey};
-    background-color: ${theme.colour.white};
-    padding: ${theme.spacing.sm} 0;
-    text-align: center;
+    font-size: ${theme.font.md};
+    color: ${theme.colour.white};
 
-    ${mediaQuery.sm(css`
-      width: 13em;
+    ${mediaQuery.lg(css`
+      width: 100% !important;
+      color: ${theme.colour.black};
+      font-size: ${theme.font.md};
     `)};
 
     &.empty {
-      background-color: ${theme.colour.greyLight};
-      border: 2px solid ${theme.colour.greyLight};
-
       * {
-        visibility: hidden;
+        display: none;
```

I was looking into this, whether setting an error message from `display:none` to `display:block` was a good idea.

I haven't found what looks like the definitive answer.

- [this article](http://web-accessibility.carnegiemuseums.org/code/alerts/) says to use `display:none` to hide error messages until we need them
- [this article](http://www.davidmacd.com/blog/test-aria-live-display-none.html) and [this one](https://developer.paciellogroup.com/blog/2012/06/html5-accessibility-chops-aria-rolealert-browser-support/) are both saying that it might not work on Safari

Let's keep your CSS but then we'll run this manually / see if the accessibility review picks this up.